main.go

```diff
@@ -54,18 +56,20 @@ func main() {
 
 	// Set the before function.
 	p.Before = func(ctx context.Context) error {
+		tokenFile = path.Join(os.TempDir(), "token.json")
```

Comment:
can it be filepath.Join :)

Reply:
oh and this can just be the default in the flag line :)

Reply:
Changes have been made. 

I've also used the `/tmp/token.json` as default value in the README since I guess this makes more sense for most users.